<root>
    <TreeNodesModel>
        <Action ID="CompareEpickObjectDetectionStatus">
            <metadata subcategory="Robotiq Epick Gripper"/>
            <description>
                <p>
                    Compare two ObjectDetectionStatus messages.
                </p>
            </description>
            <input_port name="value1" default="">First ObjectDetectionStatus message to compare.</input_port>
            <input_port name="value2" default="">Second ObjectDetectionStatus message to compare.</input_port>
        </Action>
    </TreeNodesModel>
    <TreeNodesModel>
        <Action ID="GetEpickObjectDetectionStatus">
            <metadata subcategory="Robotiq Epick Gripper"/>
            <description>
                <p>
                    Captures an ObjectDetectionStatus message and makes it available on an output port.
                </p>
            </description>
            <input_port name="topic_name" default="">ObjectDetectionStatus topic the behavior subscribes to.</input_port>
            <output_port name="message_out" default="{point_cloud}">Contains the message that was received on the topic.</output_port>
        </Action>
    </TreeNodesModel>
</root>
